CHANGELOG.md

```diff
@@ -78,6 +78,11 @@
   [Martin Redington](https://github.com/mildm8nnered)
   [#5137](https://github.com/realm/SwiftLint/issues/5137)
 
+* The `no_magic_numbers` rule will not trigger for the either operand of
```

Comment:
nit: "...rule will not trigger for bitwise shift operations"

Reply:
nice catch - thanks! I've cleaned it up